### DISCUSSION TITLE
Is anyone still working on the project?

### DISCUSSION THREAD

**Discussion by bot17** [Sn0w3y/NEP-Inverter-Reversing#13] [General]
If so, can anyone tell me whether the power values that are transmitted are correct? The cloud does not seem to be outputting the values correctly. Or in other words, since the values transmitted by the inverter (every 5 minutes) are not processed correctly in the cloud, are they transmitted correctly by the inverter? 

![image](https://github.com/user-attachments/assets/2f325aa9-e0aa-4f78-a56e-93e309a05bc9)

I think the data should be in the same format as in the NEPviewerCR history? 

So there’s Date,Time, kW, MPPT/DC voltage, AC voltage, AC amps, AC frequency, temperature, total kWh, RSSI maybe, mystery and error code.

**Comment by Sn0w3y** [comment]
you are VERY welcome to provide a PR ! :)

**Comment by bot17** [comment]
was just a question. Perhaps the data could also be read directly from the inverter? Like in the web_ui_update. What is app_upgrade_original actually for?

**Comment by Sn0w3y** [comment]
> are not processed correctly in the cloud

basically they are - they just average the values in some weird way i still do not understand..

I will sort your suggestion out if I have time for it :)

That was the reason why i asked you for a PR :D There was no bad intention behind it :)

**Comment by bot17** [comment]
Everything is fine.
What I mean by not correct are the power values (watts) in the cloud, I use a storage system, when it switches on early it feeds in at least 65 watts, then depending on the PV power 90, 130 and with enough power everything the house needs. 
But there is never a curve like with the module output. But the cloud always displays the data like this. See picture:

![image](https://github.com/user-attachments/assets/f286f3df-0cff-4c8c-b8b6-0b875eb52edb)

History is available at the URL: http://user.nepviewer.com/pv_monitor/proxy/history/XXXSNXXX/20241125/20241125/0/1/2  

The data is described here:
https://github.com/DrFrankReade/ReneSola_MRG

The kwH seem to fit, but you can't use this history because the cloud ignores / forgets so many values. In any case, the inverter sends them correctly every 5 minutes (checked with tcpdump on the router). 

The best thing would of course be http://inverter_ip/status and you get all the current values :-)

**Reply by bot17** [reply]
3-4 days ago something was apparently changed in the cloud. The watts from the inverter are now also displayed correctly in the morning. 
Unfortunately, the ignored values have not changed, especially at midday.
